invenio_accounts/models.py

```diff
@@ -87,3 +89,17 @@ def validate_ip(self, key, value):
         if value == 'untrackable':  # pragma: no cover
             value = None
         return value
+
+
+class SessionActivity(db.Model, Timestamp):
+    """User Session Activity model."""
+
+    __tablename__ = "accounts_user_session_activity"
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    user_id = db.Column(db.Integer, db.ForeignKey(User.id))
+
+    sid_s = db.Column(db.String(255), unique=True, index=True)
```

Comment:
@jirikuncar: docstrings as in sphinx docstrings for the model/Class, or just some inline comments?